### PR TITLE
Mac image match bug

### PR DESCRIPTION
**Pull request by evrose54**
# Reviewer Checklist

* [ ] Required ***existing tests*** pass (ie full_test.sh, others as appropriate)
* [ ] NO REQUIRED ***unit tests*** (explain why not required)
* [ ] NO REQUIRED ***integration tests*** (explain why not required)
* [ ] Required ***documentation*** added for new/modified functionality
* [ ] Required ***release notes*** added for new/modified functionality
* [ ] NO REQUIRED ***updates to other repos*** (explain why not required)

# Related Issues
No issues for this PR.

# Testing Instructions
Run ``./tests/integration_tests/full_test.sh`` and get an output of 0.

# Summary
While testing Mac for the 2024 tutorial, it was found quite a few tests were
failing due to bad image comparisons. Upon greater inspection, it was found that
these tests were failing only due to a couple of different pixels, which we assume
is due to different backend mechanics depending on your OS. Instead of failing a
test if a single pixel is outside of a certain threshold, we've decided to test an
image's match with another pixel via a group basis.

Ie. If num_pixels_mismatched_by_threshold / total_num_pixels > x%, then we'll
raise an error. This is a more lenient test, but will still catch differences
between two images. All in all, we wan't to ignore tiny differences between images
and catch the more glaring differences that affect lots of pixels when they occur.

    modified: geoips/plugins/modules/output_checkers/image.py

### CODE REVIEW

File: environments/mamba_full_package_list_1.13.0_20240724.yml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+# $ conda create --name <env> --file <this file>
+# platform: linux-64
+_libgcc_mutex=0.1=conda_forge
+_openmp_mutex=4.5=2_gnu
+_sysroot_linux-64_curr_repodata_hack=3=h69a702a_16
+accessible-pygments=0.0.5=pypi_0
+affine=2.4.0=pypi_0
+akima86=0.1.1=pypi_0
+alabaster=0.7.16=pypi_0
+annotated-types=0.7.0=pypi_0
```
Comment (mindyls) [inline environments/mamba_full_package_list_1.13.0_20240724.yml:11]: Do we want to add these to the YAML release note?  Maybe it is good to track per-PR when we create these environment dumps?  Along with a copy/paste of the full-test output, showing that these dependency versions do in fact relate to a complete 0 return for everything?

Comment (mindyls) [reply environments/mamba_full_package_list_1.13.0_20240724.yml:11]: I kind of like tracking this, but that environemnts directory is going to get rather cluttered!!!

**Review by mindyls — APPROVED** [review]
My full test is running now, but so far it looks good

**Comment by evrose54** [comment]
This is the only remaining failing test minus ``ami.tc.WV.geotiff.sh`` which we said not to worry about. 
```
Wed Jul 24 11:36:39 MDT 2024  Running viirsclearnight.Night-Vis-IR-GeoIPS1.unprojected_image.sh
./../..//tests/scripts/viirsclearnight.Night-Vis-IR-GeoIPS1.unprojected_image.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_viirsclearnight.Night-Vis-IR-GeoIPS1.unprojected_image.sh.log
        Return: 1
```
After further inspection, there are 0.0008% of exactly mismatched pixels which is greater than the default 0.0005%. We can change this by manually changing the default threshold used in Image match, but I don't think it's worth it. Mac is weird with imagery and I say this is GTG.

File: environments/mamba_full_package_list_1.13.0_20240724.yml (hunk 1 above)
Comment (evrose54) [reply environments/mamba_full_package_list_1.13.0_20240724.yml:11]: Might not be the worst idea. I'll leave that up to you and @jsolbrig.

**Comment by evrose54** [comment]
Full test ran on Mac:
```
Wed Jul 24 11:24:05 MDT 2024 Running pre, log: /Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log

Wed Jul 24 11:24:05 MDT 2024  Running export
export GEOIPS_DISABLE_SHARED_CODE_CHECKS=False
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_export__GEOIPS_DISABLE_SHARED_CODE_CHECKS=False.log
        Return: 0

Wed Jul 24 11:24:05 MDT 2024  Running check_code.sh
./../..//tests/utils/check_code.sh all ./../../
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_check_code.sh__all_._.._.._.log
        Return: 0

Wed Jul 24 11:25:40 MDT 2024  Running export
export GEOIPS_DISABLE_SHARED_CODE_CHECKS=True
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_export__GEOIPS_DISABLE_SHARED_CODE_CHECKS=True.log
        Return: 0

Wed Jul 24 11:25:40 MDT 2024  Running check_code.sh
./../..//tests/utils/check_code.sh all /Users/erose/geoips/geoips_packages/recenter_tc
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_check_code.sh__all__Users_erose_geoips_geoips_packages_recenter_tc.log
        Return: 0

Wed Jul 24 11:25:41 MDT 2024  Running build_docs.sh
./../..//docs/build_docs.sh ./../../ geoips html_only
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_build_docs.sh__._.._..__geoips_html_only.log
        Return: 0

Wed Jul 24 11:26:07 MDT 2024  Running build_docs.sh
./../..//docs/build_docs.sh /Users/erose/geoips/geoips_packages/recenter_tc recenter_tc html_only
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_build_docs.sh___Users_erose_geoips_geoips_packages_recenter_tc_recenter_tc_html_only.log
        Return: 1

Wed Jul 24 11:26:07 MDT 2024  Running build_docs.sh
./../..//docs/build_docs.sh /Users/erose/geoips/geoips_packages/data_fusion data_fusion html_only
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_build_docs.sh___Users_erose_geoips_geoips_packages_data_fusion_data_fusion_html_only.log
        Return: 0

Wed Jul 24 11:26:11 MDT 2024  Running build_docs.sh
./../..//docs/build_docs.sh /Users/erose/geoips/geoips_packages/template_basic_plugin my_package html_only
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_build_docs.sh___Users_erose_geoips_geoips_packages_template_basic_plugin_my_package_html_only.log
        Return: 0

Wed Jul 24 11:26:14 MDT 2024  Running abi.static.Infrared.imagery_annotated.sh
./../..//tests/scripts/abi.static.Infrared.imagery_annotated.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_abi.static.Infrared.imagery_annotated.sh.log
        Return: 0

Wed Jul 24 11:26:36 MDT 2024  Running console_script_create_sector_image.sh
./../..//tests/scripts/console_script_create_sector_image.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_console_script_create_sector_image.sh.log
        Return: 0

Wed Jul 24 11:26:39 MDT 2024  Running console_script_list_available_plugins.sh
./../..//tests/scripts/console_script_list_available_plugins.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_console_script_list_available_plugins.sh.log
        Return: 0

Wed Jul 24 11:26:47 MDT 2024  Running abi.static.Visible.imagery_annotated.sh
./../..//tests/scripts/abi.static.Visible.imagery_annotated.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_abi.static.Visible.imagery_annotated.sh.log
        Return: 0

Wed Jul 24 11:27:12 MDT 2024  Running abi.config_based_output_low_memory.sh
./../..//tests/scripts/abi.config_based_output_low_memory.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_abi.config_based_output_low_memory.sh.log
        Return: 0

Wed Jul 24 11:27:52 MDT 2024  Running abi.config_based_output.sh
./../..//tests/scripts/abi.config_based_output.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_abi.config_based_output.sh.log
        Return: 0

Wed Jul 24 11:28:39 MDT 2024  Running ahi.tc.WV.geotiff.sh
./../..//tests/scripts/ahi.tc.WV.geotiff.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ahi.tc.WV.geotiff.sh.log
        Return: 0

Wed Jul 24 11:28:44 MDT 2024  Running ami.static.Infrared.imagery_annotated.sh
./../..//tests/scripts/ami.static.Infrared.imagery_annotated.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ami.static.Infrared.imagery_annotated.sh.log
        Return: 0

Wed Jul 24 11:29:06 MDT 2024  Running ami.static.Visible.imagery_annotated.sh
./../..//tests/scripts/ami.static.Visible.imagery_annotated.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ami.static.Visible.imagery_annotated.sh.log
        Return: 0

Wed Jul 24 11:30:15 MDT 2024  Running ami.tc.WV.geotiff.sh
./../..//tests/scripts/ami.tc.WV.geotiff.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ami.tc.WV.geotiff.sh.log
        Return: 1

Wed Jul 24 11:30:19 MDT 2024  Running ami.WV-Upper.unprojected_image.sh
./../..//tests/scripts/ami.WV-Upper.unprojected_image.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ami.WV-Upper.unprojected_image.sh.log
        Return: 0

Wed Jul 24 11:30:38 MDT 2024  Running amsr2.tc.89H-Physical.imagery_annotated.sh
./../..//tests/scripts/amsr2.tc.89H-Physical.imagery_annotated.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_amsr2.tc.89H-Physical.imagery_annotated.sh.log
        Return: 0

Wed Jul 24 11:30:57 MDT 2024  Running amsr2_ocean.tc.windspeed.imagery_clean.sh
./../..//tests/scripts/amsr2_ocean.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_amsr2_ocean.tc.windspeed.imagery_clean.sh.log
        Return: 0

Wed Jul 24 11:31:05 MDT 2024  Running amsr2.config_based_no_compare.sh
./../..//tests/scripts/amsr2.config_based_no_compare.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_amsr2.config_based_no_compare.sh.log
        Return: 0

Wed Jul 24 11:31:41 MDT 2024  Running amsr2.config_based_overlay_output.sh
./../..//tests/scripts/amsr2.config_based_overlay_output.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_amsr2.config_based_overlay_output.sh.log
        Return: 0

Wed Jul 24 11:32:54 MDT 2024  Running amsr2.config_based_overlay_output_low_memory.sh
./../..//tests/scripts/amsr2.config_based_overlay_output_low_memory.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_amsr2.config_based_overlay_output_low_memory.sh.log
        Return: 0

Wed Jul 24 11:33:55 MDT 2024  Running ascat_knmi.tc.windbarbs.imagery_windbarbs_clean.sh
./../..//tests/scripts/ascat_knmi.tc.windbarbs.imagery_windbarbs_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_knmi.tc.windbarbs.imagery_windbarbs_clean.sh.log
        Return: 0

Wed Jul 24 11:34:05 MDT 2024  Running ascat_low_knmi.tc.windbarbs.imagery_windbarbs.sh
./../..//tests/scripts/ascat_low_knmi.tc.windbarbs.imagery_windbarbs.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_low_knmi.tc.windbarbs.imagery_windbarbs.sh.log
        Return: 0

Wed Jul 24 11:34:18 MDT 2024  Running ascat_noaa_25km.tc.windbarbs.imagery_windbarbs.sh
./../..//tests/scripts/ascat_noaa_25km.tc.windbarbs.imagery_windbarbs.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_noaa_25km.tc.windbarbs.imagery_windbarbs.sh.log
        Return: 0

Wed Jul 24 11:34:32 MDT 2024  Running ascat_noaa_50km.tc.wind-ambiguities.imagery_windbarbs.sh
./../..//tests/scripts/ascat_noaa_50km.tc.wind-ambiguities.imagery_windbarbs.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_noaa_50km.tc.wind-ambiguities.imagery_windbarbs.sh.log
        Return: 0

Wed Jul 24 11:34:44 MDT 2024  Running ascat_uhr.tc.wind-ambiguities.imagery_windbarbs.sh
./../..//tests/scripts/ascat_uhr.tc.wind-ambiguities.imagery_windbarbs.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_uhr.tc.wind-ambiguities.imagery_windbarbs.sh.log
        Return: 0

Wed Jul 24 11:34:58 MDT 2024  Running ascat_uhr.tc.nrcs.imagery_clean.sh
./../..//tests/scripts/ascat_uhr.tc.nrcs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_uhr.tc.nrcs.imagery_clean.sh.log
        Return: 0

Wed Jul 24 11:35:04 MDT 2024  Running ascat_uhr.tc.windbarbs.imagery_windbarbs.sh
./../..//tests/scripts/ascat_uhr.tc.windbarbs.imagery_windbarbs.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_uhr.tc.windbarbs.imagery_windbarbs.sh.log
        Return: 0

Wed Jul 24 11:35:24 MDT 2024  Running ascat_uhr.tc.windspeed.imagery_clean.sh
./../..//tests/scripts/ascat_uhr.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_uhr.tc.windspeed.imagery_clean.sh.log
        Return: 0

Wed Jul 24 11:35:30 MDT 2024  Running cli_dummy_script.sh
./../..//tests/scripts/cli_dummy_script.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_cli_dummy_script.sh.log
        Return: 0

Wed Jul 24 11:35:30 MDT 2024  Running gmi.tc.89pct.imagery_clean.sh
./../..//tests/scripts/gmi.tc.89pct.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_gmi.tc.89pct.imagery_clean.sh.log
        Return: 0

Wed Jul 24 11:35:38 MDT 2024  Running imerg.tc.Rain.imagery_clean.sh
./../..//tests/scripts/imerg.tc.Rain.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_imerg.tc.Rain.imagery_clean.sh.log
        Return: 0

Wed Jul 24 11:35:46 MDT 2024  Running oscat_knmi.tc.windbarbs.imagery_windbarbs.sh
./../..//tests/scripts/oscat_knmi.tc.windbarbs.imagery_windbarbs.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_oscat_knmi.tc.windbarbs.imagery_windbarbs.sh.log
        Return: 0

Wed Jul 24 11:35:59 MDT 2024  Running sar.tc.nrcs.imagery_annotated.sh
./../..//tests/scripts/sar.tc.nrcs.imagery_annotated.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_sar.tc.nrcs.imagery_annotated.sh.log
        Return: 0

Wed Jul 24 11:36:09 MDT 2024  Running smap.unsectored.text_winds.sh
./../..//tests/scripts/smap.unsectored.text_winds.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_smap.unsectored.text_winds.sh.log
        Return: 0

Wed Jul 24 11:36:13 MDT 2024  Running viirsday.tc.Night-Vis-IR.imagery_annotated.sh
./../..//tests/scripts/viirsday.tc.Night-Vis-IR.imagery_annotated.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_viirsday.tc.Night-Vis-IR.imagery_annotated.sh.log
        Return: 0

Wed Jul 24 11:36:30 MDT 2024  Running viirsmoon.tc.Night-Vis-GeoIPS1.imagery_clean.sh
./../..//tests/scripts/viirsmoon.tc.Night-Vis-GeoIPS1.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_viirsmoon.tc.Night-Vis-GeoIPS1.imagery_clean.sh.log
        Return: 0

Wed Jul 24 11:36:39 MDT 2024  Running viirsclearnight.Night-Vis-IR-GeoIPS1.unprojected_image.sh
./../..//tests/scripts/viirsclearnight.Night-Vis-IR-GeoIPS1.unprojected_image.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_viirsclearnight.Night-Vis-IR-GeoIPS1.unprojected_image.sh.log
        Return: 1

Wed Jul 24 11:37:15 MDT 2024  Running abi.tc.Visible.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/abi.tc.Visible.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_abi.tc.Visible.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:20 MDT 2024  Running amsr2.tc.color37.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/amsr2.tc.color37.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_amsr2.tc.color37.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:28 MDT 2024  Running amsr2.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/amsr2.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_amsr2.tc.windspeed.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:31 MDT 2024  Running ascat_uhr.tc.nrcs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/ascat_uhr.tc.nrcs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_uhr.tc.nrcs.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:33 MDT 2024  Running ascat_uhr.tc.windbarbs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/ascat_uhr.tc.windbarbs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_ascat_uhr.tc.windbarbs.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:35 MDT 2024  Running imerg.tc.Rain.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/imerg.tc.Rain.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_imerg.tc.Rain.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:37 MDT 2024  Running metopc_knmi_125.tc.windbarbs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/metopc_knmi_125.tc.windbarbs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_metopc_knmi_125.tc.windbarbs.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:38 MDT 2024  Running oscat.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/oscat.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_oscat.tc.windspeed.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:40 MDT 2024  Running sar.tc.nrcs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/sar.tc.nrcs.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_sar.tc.nrcs.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:41 MDT 2024  Running smap.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/smap.tc.windspeed.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_smap.tc.windspeed.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:43 MDT 2024  Running viirs.tc.Infrared-Gray.imagery_clean.sh
/Users/erose/geoips/geoips_packages/recenter_tc/tests/scripts/viirs.tc.Infrared-Gray.imagery_clean.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_viirs.tc.Infrared-Gray.imagery_clean.sh.log
        Return: 1

Wed Jul 24 11:37:55 MDT 2024  Running test_all.sh
/Users/erose/geoips/geoips_packages/template_basic_plugin/tests/test_all.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_test_all.sh.log
        Return: 0

Wed Jul 24 11:39:09 MDT 2024  Running test_all.sh
/Users/erose/geoips/geoips_packages/geoips_plugin_example/tests/test_all.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_test_all.sh.log
        Return: 0

Wed Jul 24 11:39:28 MDT 2024  Running test_all.sh
/Users/erose/geoips/geoips_packages/geoips_clavrx/tests/test_all.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_test_all.sh.log
        Return: 0

Wed Jul 24 11:43:16 MDT 2024  Running test_all.sh
/Users/erose/geoips/geoips_packages/data_fusion/tests/test_all.sh
/Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log_test_all.sh.log
        Return: 0


Wed Jul 24 11:45:16 MDT 2024  Running post, final results in /Users/erose/geoips/geoips_packages/outdirs/logs/20240724/20240724.172405_geoips_full/test_all_geoips_full.log

Package: geoips_full
Total run time: 1271 seconds
Number data types run: 56
Number data types failed: 14
Wed Jul 24 17:45:16 UTC 2024
```

**Review by cpcamacho — APPROVED** [review]

File: environments/mamba_full_package_list_1.13.0_20240724.yml (hunk 1 above)
Comment (jsolbrig) [reply environments/mamba_full_package_list_1.13.0_20240724.yml:11]: I say we discuss this later and just get this merged as is.

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**